examples/user_profiles/connector.py

```diff
@@ -0,0 +1,144 @@
+from fivetran_connector_sdk import Connector
+from fivetran_connector_sdk import Operations as op
+from fivetran_connector_sdk import Logging as log
+import requests as rq
+import pandas as pd
+
+
+def schema (configuration: dict):
+    return [
+        {
+            "table": "profile",
+            "primary_key": ["id"],
+            "columns": {
+                "id": "INT",
+                "gender": "STRING",
+                "fullName": "STRING",
+                "email": "STRING",
+                "age": "INT",
+                "date": "STRING",
+                "mobile": "STRING",
+                "nationality": "STRING"
+            }
+
+        },
+        {
+            "table": "location",
+            "primary_key": ["profileId"],
+            "columns": {
+                "profileId": "INT",
+                "street": "STRING",
+                "city": "STRING",
+                "state": "STRING",
+                "country": "STRING",
+                "postcode": "STRING"
+            }
+        },
+        {
+            "table": "Login",
+            "primary_key": ["profileId"],
+            "columns": {
+                "profileId": "INT",
+                "uuid": "STRING",
+                "username": "STRING",
+                "password": "STRING"
+            }
+        }
+    ]
+
+
+def update(configuration: dict, state: dict):
+    profile_cursor = state["profile_cursor"] if "profile_cursor" in state else 0
+
+    profile_df, location_df, login_df, cursor = get_data(profile_cursor)
+
+    # UPSERT all profile table data and checkpointing after every 5 records
```

Comment:
comments are not consistent, please fix them.

Reply:
No need to add comment if the code is clear. If the code is not clear, let's make it clear

Reply:
Added verbose comments to example.